### PR TITLE
Fixed links and a bit more

### PR DESCRIPTION
**Pull request by Piterden**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -160,7 +161,7 @@ Context is created per request and contains following props:
 
 ```js
 bot.use((ctx) => {
-  ctx.telegram             // Telegram client instance
+  ctx.tg                   // Telegram client instance
```
Comment (dotcypress) [inline docs/README.md]: We have [public accessor](https://github.com/telegraf/telegraf/blob/develop/core/context.js#L62) for Telegram instance.

**Comment by Piterden** [comment]
Hm... Okay. But why not to do `this.telegram = telegram` there
https://github.com/telegraf/telegraf/blob/develop/core/context.js#L43
???

**Comment by dotcypress** [comment]
Because this is argument with name 'telegram'

**Comment by Piterden** [comment]
Sorry, I am not follow.
So, an argument named `telegram` passed to the constructor, and what?

**Pull request merged**